guides/v2.1/migration/migration-overview-how.md

```diff
@@ -4,41 +4,64 @@ title: How migration works
 redirect_from: /guides/v1.0/migration/migration-overview-how.html
```

Comment:
It's OK to delete redirects that point to the `v1.0` directory. It's an artifact from the pre 2.0 days (2015) and Google won't be serving up those very stale links.

Reply:
Good to know. Will delete.